sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/complexTypeCreator.scala

```diff
@@ -325,7 +325,7 @@ object CreateStruct extends FunctionBuilder {
    */
   val registryEntry: (String, (ExpressionInfo, FunctionBuilder)) = {
     val info: ExpressionInfo = new ExpressionInfo(
-      "org.apache.spark.sql.catalyst.expressions.NamedStruct",
```

Comment:
Could you use `setAlias` in expression instead? I think `struct` is a alias of `named_struct`. cc: @HyukjinKwon @hvanhovell 
https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala#L581

Reply:
+1 we can additionally override `sql` and show  the parameters pretty as well, e.g.) `named_struct(col1, 1, col2, 2)` -> `struct(1, 2)`